client/src/app/core/users/user.model.ts

```diff
@@ -149,4 +149,21 @@ export class User implements UserServerModel {
   updateAccountAvatar (newAccountAvatar: Avatar) {
     this.account.updateAvatar(newAccountAvatar)
   }
+
+  isUploadDisabled (): boolean {
+    return (this.videoQuota === 0 || this.videoQuotaDaily === 0)
+  }
+
+  canSeeVideosLink (): boolean {
+    if (!this.isUploadDisabled()) {
+      return true
+    }
+
+    // cannot upload but has already some videos
+    if (this.videosCount > 0) {
```

Comment:
@Chocobozzz we have an issue here, it seems `videosCount` is always undefined in this context

Reply:
Because we don't `videosCount` in the `/me` endpoint. I'll update this PR to inject it